elementary/monitor/dbt_project/macros/alerts/get_test_alerts.sql

```diff
@@ -122,7 +124,8 @@
     {% set select_last_alert_sent_times_query %}
         with alerts_in_time_limit as (
             select
-                test_unique_id,
+                {# Generate elementary unique id which is used to identify between tests, and set it as unique_id #}
```

Comment:
Because test id is shared between tests, we need to use elementary unique id (like we have on current test results query) to identify between tests.
We use this unique id to dedup alerts.